recipes/ecco_v4_py/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.1.6" %}
+
+package:
+  name: ecco_v4_py
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/e/ecco_v4_py/ecco_v4_py-{{ version }}.tar.gz
+  sha256: 419da92d36a739586f31c9b0bc132c1fd91b8cb33b5ef58064f64b2caa4a954b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - future
+    - pathlib
+    - numpy
+    - matplotlib
+    - cartopy
+    - dask
+    - xarray
+    - pyresample
+    - python-dateutil
+    - xmitgcm
+    - xgcm
+
+
+test:
+  imports:
+    - ecco_v4_py
+    - ecco_v4_py.test
+  requires:
```

Comment:
Unless you are actually running tests, you don't need what is in `requires` to import the module and submodule

Reply:
Gotcha. We probably don't need to run the tests on installation, should I just remove the whole section?

Reply:
Importing `ecco_v4_py.test` doesn't actually run any of the tests. It's actually a no-op since there is nothing in the `__init__.py`. It ends up just being an existence test, which is fine to leave.

Reply:
Thanks, that makes sense. I will remove it for now because actually the version on pypi is before the test module was there. So many things TBD...